scripts/ForwarderDeployer.js

```diff
@@ -1,25 +1,47 @@
 
-
+//todo
+//rename to biconomy forwarder and fee proxy (erc20 forwarder) deployer
+//Kovan
 async function main() {
     let daiEthPriceFeedAddress = "0x22B58f1EbEDfCA50feF632bD73368b2FdA96D541";
     let daiAddress = "0x4F96Fe3b7A6Cf9725f59d353F723c1bDb64CA6Aa";
-    let decimals = 18;
+    let daiDecimals = 18;
+
+    let usdtEthPriceFeedAddress ="0x0bF499444525a23E7Bb61997539725cA2e928138";
+    let usdtAddress = "0x8486E3592CC23F27Ea430aDAD031Cf8ABD978756";
+    let usdtDecimals = 6;
+
+    let usdcEthPriceFeedAddress = "0x64EaC61A2DFda2c3Fa04eED49AA33D021AeC8838";
+    let usdcAddress = "0xA5d2a43d94B6a26054506D18ea44E4749f727721";
+    let usdcDecimals = 18;
 
     const accounts = await hre.ethers.getSigners();
+    
     const Forwarder = await hre.ethers.getContractFactory("BiconomyForwarder");
     const forwarder = await Forwarder.deploy();
-
     await forwarder.deployed();
     console.log("Biconomy Forwarder deployed at : ",forwarder.address);
-    await forwarder.registerDomainSeparator("TEST","1");
+    await forwarder.registerDomainSeparator("TRUSTED FORWARDER","1");
```

Comment:
Use "Biconomy Forwarder" as name

Reply:
ok. Changing this in the configs also. 
all caps?